src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Textbox from '../textbox';
+import I18nHelper from '../../../utils/helpers/i18n';
+import Logger from '../../../utils/logger';
+
+class Decimal extends React.Component {
+  constructor(props) {
+    super(props);
+    const { value } = this.props;
+    const isValidInitVal = this.testValue(value);
+    const initVal = isValidInitVal ? value : Decimal.defaultProps.value;
```

Comment:
should we fallback to the default if they're providing a dodgy value? or throw an error? if so this could be prop type validation